portlets/src/main/webapp/vue-app/engagement-center/components/challenges/ChallengeCard.vue

```diff
@@ -171,7 +171,7 @@ export default {
         const days = Math.round((this.startDate.getTime() - new Date().getTime()) / (1000 * 60 * 60 * 24)) + 1;
         return this.$t('challenges.label.openIn', {0: days});
       } else {
-        const days = Math.round((this.endDate.getTime() - this.startDate.getTime()) / (1000 * 60 * 60 * 24)) + 1;
+        const days = Math.round((this.endDate.getTime() - new Date().getTime()) / (1000 * 60 * 60 * 24)) + 1;
```

Comment:
please replace dates comparaison in previous `if else`, the date comparaison should be done using timestamp comparaison, try `(new Date() > new Date()) || (new Date() < new Date()) || (new Date() == new Date())` it never givers `true`

Reply:
cc @boubaker 
When I try to put this code, an eslint error appears "no-self-compare"

![image](https://user-images.githubusercontent.com/40363501/218707590-09456dd9-7e37-4524-9b70-b0712634036b.png)

Reply:
I didn't meant to add the code inside the file, it's just to indicate that the dates comparaison doesn't work and you should user Date.getTime for timestamp comparaison. The given code is to test on browser console if you want to see that it never gives true as said.

Reply:
cc @boubaker done!
I misunderstood what you said, sorry!

Reply:
no worry, thanks for considering